src/components/content/myServices/MyServices.tsx

```diff
@@ -246,15 +246,15 @@ function MyServices(): React.JSX.Element {
                                     icon={<CloseCircleOutlined />}
                                     disabled={true}
                                 >
-                                    destroy
+                                    purge
```

Comment:
we must not rename the destroy button here to purge. We need both purge and destroy functionalities based on order states.